### ISSUE TITLE
Consider using NuGet's buildTransitive folder

### ISSUE THREAD

**Issue by bricelam** [dotnet/efcore#14531]
See [Allow package authors to define build assets transitive behavior](https://github.com/NuGet/Home/wiki/Allow-package--authors-to-define-build-assets-transitive-behavior).

We have some MSBuild assets in EFCore.Design. We should weigh the advantages and disadvantages of making them transitive.

We should also take another look at the way dotnet-ef reads project metadata to see if we can improve it using this feature.

**Comment by bricelam** [comment]
I don't think this is relevant after PR #34574. AFAIK, there are no MSBuild props/targets files anymore. Feel free to close if you can't think of any either. (Some may have been added for AoT)

**Comment by AndriySvyryd** [comment]
Closing this for now
